experimental/unitsconversion/src/provider.rs

```diff
@@ -68,7 +67,7 @@ pub struct UnitsInfoV1<'data> {
 pub struct ConversionInfo<'data> {
     /// Contains the base unit which the unit is converted to.
     #[cfg_attr(feature = "serde", serde(borrow))]
-    pub base_unit: Cow<'data, str>,
+    pub base_unit: ZeroVec<'data, MeasureUnitItem>,
```

Comment:
nit: introduce an alias or a newtype for this. It's confusing that a singular unit is a zerovec.

Reply:
how about `parsed_base_unit` ?

Reply:
that's still singular

Reply:
then call this `base_units`?

Reply:
but it is one unit, in a parsed form :)

Reply:
I think `basic_units` is good enough for a name, which indicate the bases of the base units.